tests/sentry/test_event_manager.py

```diff
@@ -131,8 +131,8 @@ def test_interface_is_relabeled(self):
         manager = EventManager(self.make_event(user={'id': '1'}))
         data = manager.normalize()
 
-        assert data['sentry.interfaces.User'] == {'id': '1'}
-        assert 'user' not in data
+        assert data['user'] == {'id': '1'}
+        assert 'sentry.interfaces.User' not in data.keys()
```

Comment:
Why is `keys()` necessary here? This is an odd statement and if it has a purpose a comment should clarify what it checks against the new list instead of the container itself.

Reply:
Because `__contains__` will say true if you ask `data` directly.

Reply:
Can you add a comment clarifying this? This will likely to be picked up by a linter sooner or later and someone is going to "fix" it.